src/ralph/export_to_ng/resources.py

```diff
@@ -30,11 +30,29 @@
     LicenceUser,
     SoftwareCategory,
 )
+from ralph_assets.models_dc_assets import DataCenter
+from ralph_assets.models_dc_assets import Rack as AssetRack
 
 
 logger = logging.getLogger(__name__)
 
 
+def get_data_center_id(network_datacenter):
+    """
+    Gets data center id for `network_datacenter`.
+    """
+    try:
+        data_center_id = DataCenter.objects.get(
+            name__iexact=network_datacenter.name
```

Comment:
couldn't you use `deprecated_ralph_dc` field (https://github.com/allegro/ralph_assets/blob/develop/src/ralph_assets/models_dc_assets.py#L128)?